src/Microsoft.Diagnostics.Monitoring.WebApi/ILogsOperationFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.EventPipe;
+using Microsoft.Diagnostics.Monitoring.Options;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    /// <summary>
+    /// Factory for creating operations that produce logs artifacts.
+    /// </summary>
+    internal interface ILogsOperationFactory
```

Comment:
Do we need this to be an interface?

Reply:
I want this to be implemented in dotnet-monitor, so we need an interface in WebApi for that project to call it.